### PR TITLE
SDOR-117: Some README.md corrections

### PR DESCRIPTION
**Pull request by artemuskov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # AWS_Demo_project
 
 Directories description:
-- app - Application code
+- CloudFormation - Deploy infractracture
```
Comment (babinskiy) [inline README.md]: Why our deploy infrastructure folder have so unclear name? is it ok?

Comment (artemuskov) [reply README.md]: Feel free to suggestions. Think, it's doesn't matter now

Comment (babinskiy) [reply README.md]: It may matter on presentation

**Comment by babinskiy** [comment]
good job

**Pull request merged**